examples/calculations/Sounding_Calculations.py

```diff
@@ -0,0 +1,210 @@
+kindex = mpcalc.k_index(p, T, Td)
+showalter = mpcalc.showalter_index(p, T, Td)
+total_totals = mpcalc.total_totals_index(p, T, Td)
+vert_totals = mpcalc.vertical_totals(p, T)
+
+###########################################
+# Compture the parcel profile for a surface-based parcel
+prof = mpcalc.parcel_profile(p, T[0], Td[0])
+
+###########################################
+# Compute the corresponding LI, CAPE, CIN values for a surface parcel
+lift_index = mpcalc.lifted_index(p, T, prof)
+cape, cin = mpcalc.cape_cin(p, T, Td, prof)
+
+###########################################
+# Determine the LCL, LFC, and EL for our surface parcel
+lclp, lclt = mpcalc.lcl(p[0], T[0], Td[0])
+lfcp, _ = mpcalc.lfc(p, T, Td)
+el_pressure, _ = mpcalc.el(p, T, Td, prof)
+
+###########################################
+# Compute the characteristics of a mean layer parcel (50-hPa depth)
+ml_t, ml_td = mpcalc.mixed_layer(p, T, Td, depth=50 * units.hPa)
+ml_p, _, _ = mpcalc.mixed_parcel(p, T, Td, depth=50 * units.hPa)
+mlcape, mlcin = mpcalc.mixed_layer_cape_cin(p, T, prof, depth=50 * units.hPa)
+
+###########################################
+# Compute the characteristics of the most unstable parcel (50-hPa depth)
+mu_p, mu_t, mu_td, _ = mpcalc.most_unstable_parcel(p, T, Td, depth=50 * units.hPa)
+mucape, mucin = mpcalc.most_unstable_cape_cin(p, T, Td, depth=50 * units.hPa)
+
+###########################################
+# Compute the Bunkers Storm Motion vector and use to calculate the critical angle
+bunkers = mpcalc.bunkers_storm_motion(p, u, v, height)
+u_storm, v_storm = bunkers[0]
+critical_angle = mpcalc.critical_angle(p, u, v, height, u_storm, v_storm)
+
+###########################################
+# Work on the calculations needed to compute the significant tornado parameter
+
+# Estimate height of LCL in meters from hydrostatic thickness
+new_p = np.append(p[p > lclp], lclp)
+new_t = np.append(T[p > lclp], lclt)
+lcl_height = mpcalc.thickness_hydrostatic(new_p, new_t)
+
+# Compute Surface-based CAPE
+sbcape, _ = mpcalc.surface_based_cape_cin(p, T, Td)
+
+# Compute SRH, given a motion vector toward the NE at 9.9 m/s
+srh = mpcalc.storm_relative_helicity(height, u, v, depth=1 * units.km,
+                                     storm_u=u_storm, storm_v=v_storm)
+
+# Copmute Bulk Shear components and then magnitude
+ubshr, vbshr = mpcalc.bulk_shear(p, u, v, height=height, depth=6 * units.km)
+bshear = mpcalc.wind_speed(ubshr, vbshr)
+
+# Use all computed pieces to calculate the Significant Tornado parameter
+sig_tor = mpcalc.significant_tornado(sbcape, lcl_height, srh[2], bshear).to_base_units()
+
+###########################################
+# Compute the supercell composite parameter, if possible
+
+# Determine the top and bottom of the effective layer using our own function
+hbot, htop = effective_layer(p, T, Td, height, height_layer=True)
+
+# Perform the calculation of supercell composite if an effective layer exists
+if hbot:
```

Comment:
Since this example runs with fixed input data, is there really value in making this conditional?

Reply:
Just for completeness in case someone uses this code for a different set of data.